docs/pages/xy-grid.md

```diff
@@ -117,6 +117,19 @@ below that. If you want to add padding below the `$max-width`, simply add the
   </div>
 </div>
 ```
+
+If you have just a single cell for a padding grid, you can save some markup by combining the `.grid-container`, `.grid-x` and `.grid-padding-x` classes together on the same element. You can still nest more grids inside this container like usual.
```

Comment:
This should actually work for any padding grid, regardless of cells inside... the need to have the container be a separate div for the outermost container is a margin grid requirement.  This essentially comes from the fact that the container uses `margin:auto;` to center it on the page, but the margin grid itself influences margins